sql/core/src/main/scala/org/apache/spark/sql/execution/SparkSqlParser.scala

```diff
@@ -313,18 +313,48 @@ class SparkSqlAstBuilder(conf: SQLConf) extends AstBuilder {
    * Create a [[CreateTableUsing]] or a [[CreateTableUsingAsSelect]] logical plan.
    */
   override def visitCreateTableUsing(ctx: CreateTableUsingContext): LogicalPlan = withOrigin(ctx) {
-    val (table, temp, ifNotExists, external) = visitCreateTableHeader(ctx.createTableHeader)
-    if (external) {
```

Comment:
we should allow `EXTERNAL` keyword when the `path` is specified, to be consistent with hive serde table.